.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+      - next
+
+jobs:
+  semantic-release:
+    name: Build & Publish
+    if: "!contains(github.event.head_commit.message, 'SKIP CI')"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [10.x]
+    steps:
+
+      # Setup Git Repository & Node
+      #
+      - name: Checkout branch (${{ github.ref }})
+        uses: actions/checkout@master
+      - name: Configure git
+        run: |
+          git config user.name "Jon Ursenbach"
+          git config user.email "jon@readme.io"
+      - name: Setup node (v${{ matrix.node-version }})
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install packages
+        run: |
+          npm ci
+          npx lerna bootstrap --concurrency 2
+        env:
+          CI: true
+
+      # Version, build, tag, and publish the new release;
+      #
+      - name: Build new release
```

Comment:
@rafegoldberg So the current flow for building a new Explorer release is to run:

```
npm run build
npm run deploy
npm run publish // this runs `npx lerna publish`
```

I guess for  the `build && deploy` stage that should be its own step before "Build new release", but any ideas on how or where `npx lerna publish` should be run to update the individual packages?

Reply:
Semantic release should automatically run the `npm publish` command at the end of the process!